src/renderer/components/SplittedChatListAndView.js

```diff
@@ -41,6 +41,9 @@ class SplittedChatListAndView extends React.Component {
     this.onCreateContact = this.onCreateContact.bind(this)
     this.initiateKeyTransfer = this.initiateKeyTransfer.bind(this)
     this.onKeyTransferComplete = this.onKeyTransferComplete.bind(this)
+    this.onWindowFocus = this.onWindowFocus.bind(this)
+
+    window.onfocus = this.onWindowFocus
```

Comment:
By using `onfocus`, is the message only marked noticed when you click on the ChatView window? The behavior of `onfocus` in the window seems a bit dubious, perhaps 

Maybe it would cover more edge cases if this happened instead on componentDidUpdate where chatId is different than the previous chatId? I think that will catch anytime the chatview is updated with a new chat id.

Reply:
onfocus is obsolete, changed the behaviour to allways select the chat as noticed if its the selected one, also on rerender. So if new messages come in the selected chat,they are always noticed